test/integration/community/serverspec/daemons_spec.rb

```diff
@@ -36,57 +36,57 @@
 # end
 
 describe "Alfresco daemons" do
-  let(:repoConnection) { $repoConnection ||= getFaradayConnection "http://localhost:8070" }
-  let(:shareConnection) { $shareConnection ||= getFaradayConnection "http://localhost:8081" }
-  let(:solrConnection) { $solrConnection ||= getFaradayConnection "http://localhost:8090" }
-  let(:activitiConnection) { $activitiConnection ||= getFaradayConnection "http://localhost:8060" }
-  let(:haproxyConnection) { $haproxyConnection ||= getFaradayConnection "http://localhost:9001" }
-  let(:haproxyIntConnection) { $haproxyIntConnection ||= getFaradayConnection "http://localhost:9000" }
-  let(:httpNginxConnection) { $httpNginxConnection ||= getFaradayConnection "http://localhost" }
-  let(:nginxConnection) { $nginxConnection ||= getFaradayConnection "http://#{alfresco_host}" }
-  let(:authNginxConnection) { $authNginxConnection ||= getFaradayConnection "http://admin:admin@#{alfresco_host}" }
+  # let(:repoConnection) { $repoConnection ||= getFaradayConnection "http://localhost:8070" }
```

Comment:
Can you add a comment in here explaining why these lines have been commented out?